src/manage_pg.c

```diff
@@ -212,17 +212,79 @@ manage_create_sql_functions ()
        " LANGUAGE C;",
        GVM_LIB_INSTALL_DIR);
 
-  sql ("CREATE OR REPLACE FUNCTION level_max_severity (text, text)"
-       " RETURNS double precision"
-       " AS '%s/libgvm-pg-server', 'sql_level_max_severity'"
-       " LANGUAGE C;",
-       GVM_LIB_INSTALL_DIR);
+  sql ("CREATE OR REPLACE FUNCTION level_max_severity (lvl text, cls text)"
```

Comment:
It would be good to make it clear that this must be kept the same as the C version of level_max_severity.